src/cli/mod.rs

```diff
@@ -75,6 +90,18 @@ struct LoadArgs {
     /// Arithmetic field (defaults to the backend's standard field)
     #[clap(long, value_parser)]
     field: Option<String>,
+
+    /// Path for cached public parameters
+    #[clap(long, value_parser)]
+    public_param_path: Option<PathBuf>,
+
+    /// Path for cached proofs
+    #[clap(long, value_parser)]
+    proof_path: Option<PathBuf>,
+
+    /// Path for cached commitments
+    #[clap(long, value_parser)]
+    commit_path: Option<PathBuf>,
```

Comment:
These docstrings aren't very accurate because they're not just cached. We want to save data to those directories.

Two more things:
* Let's standardize the suffix `_dir` because it's more explicit that we're talking about directories
* Let's use plurals: `public_params_dir`, `proofs_dir` and `commits_dir`